tests/test_kubernetes.py

```diff
@@ -69,20 +69,6 @@ def job_labels(request):
     return request.param
 
 
-def test_local_config_rejected(outdir, docker_image, runfile):
-    with pytest.raises(ValueError):
-        fv3config.fv3run._kubernetes._get_job(
-            "/local/dir/config.yml", outdir, docker_image, runfile
-        )
-
-
-def test_local_outdir_rejected(config_location, docker_image, runfile):
-    with pytest.raises(ValueError):
-        fv3config.fv3run._kubernetes._get_job(
-            config_location, "/local/outdir", docker_image, runfile
-        )
-
-
```

Comment:
Can you replace these with tests that they *are not* rejected?

Reply:
Hmm. It does seem like the reason for these tests no longer exists. I added a local path to `outdir` fixture so that option should be picked up by other tests.

Reply:
OK, thanks. I hadn't noticed you added the option to the outdir fixture.